code/modules/modular_computers/computers/item/computer.dm

```diff
@@ -94,6 +99,19 @@
 		card_slot.try_eject(, usr)
 
 // Eject ID card from computer, if it has ID slot with card inside.
+/obj/item/device/modular_computer/proc/eject_card()
+	set name = "Eject Intellicard"
+	set category = "Object"
+	set src in view(1)
```

Comment:
? I don't understand.
I'm basing this off of the proc for the id card which was done by @Razharas a maintainer.
Why would it be wrong?